src/databricks/labs/ucx/hive_metastore/pipelines_migrate.py

```diff
@@ -31,13 +31,13 @@ def __init__(
         pipelines_crawler: PipelinesCrawler,
         jobs_crawler: JobsCrawler,
         catalog_name: str,
-        skip_pipeline_ids: list[str] | None = None,
+        include_pipeline_ids: list[str] | None = None,
     ):
         self._ws = ws
         self._pipeline_crawler = pipelines_crawler
         self._jobs_crawler = jobs_crawler
         self._catalog_name = catalog_name
-        self._skip_pipeline_ids = skip_pipeline_ids or []
+        self._include_pipeline_ids = include_pipeline_ids or []
```

Comment:
I explain in another comment why

```suggestion
        self._include_pipeline_ids = include_pipeline_ids
```